src/layouts/Sites.tsx

```diff
@@ -163,7 +166,10 @@ export const Sites = (): JSX.Element => {
     )
   }, [siteRequestData])
 
-  const showNewLayouts = useFeatureIsOn(FEATURE_FLAGS.HOMEPAGE_TEMPLATES)
+  const { blocks } = useFeatureValue(FEATURE_FLAGS.ENABLED_BLOCKS, {
```

Comment:
Hmm why are we deviating away from a simple boolean showNewLayouts at the growthbook level and adding this layer of logic at the code level ya? is there a benefit that we are trying to achieve here?

Reply:
we need to know which blocks have been rolled out and which are not in the event of a rollback

Reply:
could we use this + site level? I think say rolling back on one site helps reduce complexity of moving parts here, thoughts?

Reply:
see above